layouts/shortcodes/dbm-sqlserver-agent-setup-windows.md

```diff
@@ -12,6 +12,10 @@ instances:
     connector: adodbapi
     adoprovider: MSOLEDBSQL
     include_ao_metrics: true  # Optional: For AlwaysOn users
+    agent_jobs:               # Optional: For monitoring SQL Server Agent jobs
+      enabled: true
+      collection_interval: 15
+      history_row_limit: 10000
```

Comment:
i'm curious if the 10k limit was ever tested ❔

Reply:
this is the default limit